### PR TITLE
🐺 E621 Command

### PR DESCRIPTION
**Pull request by Doges**
owo what's this

### CODE REVIEW

**Comment by Doges** [comment]
RFR now

**Comment by pedrofracassi** [comment]
oh of course it duped my comment

File: src/commands/nsfw/e621.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        missingError: 'commands:e621.noquery'
+      }]
+    }, client)
+  }
+
+  async run ({ t, author, channel }, query) {
+    channel.startTyping()
+    const endpoint = channel.nsfw ? '' : 'rating:safe'
+    const { posts } = await this.client.apis.e621.searchPost(`${endpoint} ${query} -flash -webm`)
+    console.log(posts)
```
Comment (perronosaurio) [inline src/commands/nsfw/e621.js:23]: ew

File: src/commands/nsfw/e621.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    try {
+      const embed = new SwitchbladeEmbed()
+      embed
+        .setImage(posts[0].file.url)
+        .setTitle(`${t('commands:e621.id')}: ${posts[0].id}`)
+        .setDescription(`**${t('commands:e621.artist')}**: "${posts[0].tags.artist[0] || 'Unknown artist'}"\n**${t('commands:e621.desc')}**: "${posts[0].description.replace(/https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/, '') || 'No Description'}"\n **ID**: ${posts[0].id}\n[${t('commands:e621.post')}](https://e621.net/posts/${posts[0].id})`)
+      channel.send(embed)
```
Comment (perronosaurio) [inline src/commands/nsfw/e621.js:30]: you can just
```suggestion
      channel.send(
        new SwitchbladeEmbed()
          .setImage(posts[0].file.url)
          .setTitle(`${t('commands:e621.id')}: ${posts[0].id}`)
          .setDescription(`**${t('commands:e621.artist')}**: "${posts[0].tags.artist[0] || 'Unknown artist'}"\n**${t('commands:e621.desc')}**: "${posts[0].description.replace(/https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/, '') || 'No Description'}"\n **ID**: ${posts[0].id}\n[${t('commands:e621.post')}](https://e621.net/posts/${posts[0].id})`)
      )
```

Comment (Doges) [reply src/commands/nsfw/e621.js:30]: its the same thing tbh

File: src/commands/nsfw/e621.js (hunk 1 above)
Comment (Doges) [reply src/commands/nsfw/e621.js:23]: forgot about it

**Pull request merged**